BackwardsCompatibility.md

```diff
@@ -0,0 +1,29 @@
+# Backward Compatibility
+The current approach to achieving backwards compatibility with EVM1 is to
+support both of the instruction sets with the option to transcompiling EVM1 to 
+EVM2. This approach gives clients optionality when dealing with EVM1 code.
+A client can either implement only a WASM VM and transcompile all of the EVM1
+code. Or a client can implement a wasm VM and EVM1 VM and leave the old code as
+is.
+
+## Gas Prices
+In eWASM we will introduce sub-gas units so that each EVM1 opcode's
+transcompiled equivalent ewasm's gas cost is less then the original EM1 opcode's
+cost. The fee schedule for eWASM is yet to be specified.
+
+## Identification of code
+We assume there is some sort of code handler function that all clients have 
+implemented. The code handler identifies the instruction Set type by whether it
+starts with WASM's magic number (0x6d736100) or not.
```

Comment:
> This magic number results in a successful execution when interpreted as EVM code.

If VM see it as [00, 61, 73, 6d] then we should be fine. Since 00 is stop. Although appended with 0xEF would be fine too if it was offically dedicate as invalid.